pkg/utils/system/system.go

```diff
@@ -0,0 +1,90 @@
+package system
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/cloud-native-application/rudrx/api/types"
+)
+
+const RudrHome = ".rudr"
```

Comment:
This is not used externally. Should make it private.

Reply:
agree